data/DiaSource.yaml

```diff
@@ -86,8 +93,42 @@ funcs:
     # trailLnL not implemented
     # trailChi2 not implemented
     # trailNdata not implemented
-    # dipMeanFlux needs functor DM-
-    # dipFluxDiff needs functor DM-
+    dipMeanFlux:
```

Comment:
"dip" is really the name of it? Not `dipole`? That's very confusing to me: I saw "dip" and thought this was about a source whose flux had gone down or something.

Reply:
Yep: http://ls.st/dpdd

Reply:
That is very unfortunate, but not a problem for this ticket.